SKILL.tsv

```diff
@@ -357,12 +357,12 @@ SKILL_20150317_000356	[Raflower] Physical, Pierce, Poison Property, Tentacle Att
 SKILL_20150317_000357	[Vubbe Wizard] Magic, Magic, Fire Property, Energy Bolt
 SKILL_20150317_000358	[Vubbe Wizard] Magic, Magic, Fire Property, Reflect Shield Buff
 SKILL_20150317_000359	[Vubbe Fire Wizard] Magic, Magic, Fire Property, Range Fireball Attack
-SKILL_20150317_000360	[Vubbe Fire Wizard] Magic, Magic, Fire Property, Fiore Pillar Attack
+SKILL_20150317_000360	[Vubbe Fire Wizard] Magic, Magic, Fire Property, Fire Pillar Attack
 SKILL_20150317_000361	[Vubbe Ice Wizard] Magic, Magic, Ice Property, Range Ice Bolt Attack
 SKILL_20150317_000362	[Vubbe Ice Wizard] Magic, Magic, Ice Property, Refreezer Wave Attack
 SKILL_20150317_000363	[Vubbe Shaman] Magic, Magic, Dark Property, Heal Floor Skill
 SKILL_20150317_000364	[Vubbe Shaman] Magic, Magic, Dark Property, Floor Skill
-SKILL_20150317_000365	[Shredded] Magic, Magic, Dark Property, Teleport to pc and Attack
+SKILL_20150317_000365	[Shredded] Magic, Magic, Dark Property, Teleport to enemy and Attack
```

Comment:
enemy -> pc 

These lines are for the developers. PC basically means you, as the player.